targets/TARGET_ublox/TARGET_R5030/gpio_api.c

```diff
@@ -93,37 +89,49 @@ void gpio_dir(gpio_t *obj, PinDirection direction)
 {
     MBED_ASSERT(obj->pin != (PinName)NC);
 
-    switch (direction)
-    {
+    switch (direction) {
       case PIN_INPUT:
     	  obj->reg_base->pio_iner_0 = obj->mask; // Enable input
     	  break;
-
       case PIN_OUTPUT:
     	  obj->reg_base->pio_oer_0 = obj->mask; // Enable output
     	  break;
-
       default:
+    	  MBED_ASSERT("Invalid direction");
```

Comment:
Correct me if I am wrong but I don't think you can print message with MBED_ASSERT. Also it may not be needed at all considering that **direction** is of type **PinDirection** so compiler would generate error if some other value is passed.

Reply:
Yes, you are right. My concept is to mention here that wrong direction for pin is selected but it is not correct way to do that. I think just mention comment here will be fine, what you say.